import_email_events.py

```diff
@@ -0,0 +1,17 @@
+#
+# Script to import "email event" metrics from S3 into redshift.
+#
+
+import import_events
+SCHEMA = """
+    type VARCHAR(30) NOT NULL,
+    locale VARCHAR(40),
+    domain VARCHAR(40),
+    template VARCHAR(64),
+    flow_id VARCHAR(64)
+"""
+
+COLUMNS = "type, locale, domain, template, flow_id"
+
+import_events.run("fxa-email", "email", SCHEMA, COLUMNS, SCHEMA, COLUMNS)
```

Comment:
Subsequent to you opening this PR originally, I updated my code to use named arguments in these calls to `run`, to make the intent clearer (as there's so many args). See the bottom of e.g. `import_activity_events` for an example.